src/common/const.go

```diff
@@ -73,6 +73,8 @@ const (
 	UAAEndpoint                = "uaa_endpoint"
 	UAAClientID                = "uaa_client_id"
 	UAAClientSecret            = "uaa_client_secret"
-
+	VerifyRemoteCert           = "verify_remote_cert"
```

Comment:
The configuration item is removed as the certificate verification is moved to the target level.